update/ansible/playbook/tasks/update.yml

```diff
@@ -22,6 +22,38 @@
       - vmproxy
       - grafana-db-migrator
   pre_tasks:
+      - name: Check if ansible-core RPM was updated
+      shell: "grep -A 1 'Install ansible-core RPM          | EL9' /srv/logs/pmm-update-perform.log | grep -qE 'changed' && echo 'Updated' || echo 'Not Updated'"
```

Comment:
since pmm-update-perform.log will keep all logs even related to past upgrades we will see this message again and again in future upgrades. can't we check `changed` task within the same run? And we don't have controllable restart, so restart might happen after we made some changes that's not supposed to be done.

Reply:
Are you certain that "pmm-update-perform.log" will retain all logs, including those pertaining to past upgrades? I initiated an upgrade via the UI, then attempted to force it again by (option + clicking) on the upgrade check button, which resulted in the replacement of the original log file.

If the log indeed preserves records of previous upgrades, then I believe we may be making this process overly complex by extracting information from the log file. That's why initially I attempted to get info from the repository itself whether an update is available for the ansible-core package.